ci/pipelines/deploy-to-production.yml

```diff
@@ -499,7 +506,6 @@ jobs:
         file: pay-ci/ci/tasks/deploy-frontend.yml
         params:
           APP_NAME: frontend
-          NGINX_IMAGE_TAG: ((.:nginx_image_tag))
```

Comment:
Do we want to use the `<<: *deploy_params` anchor for toolbox as well? I think we're still writing them out long form there?

Reply:
Yes a follow up PR will do this because Toolbox is also deploying telegraf which `<<: *deploy_params` doesn't have at the moment.

Reply:
Ah that makes sense 👍